python/paddle/fluid/framework.py

```diff
@@ -3080,13 +3082,22 @@ def find_name(var_list, name):
                     )
 
             self.desc.check_attrs()
+
+            # record all attrs needed by creating op
+            for item in self.desc.attr_names():
+                self.legacy_attrs[item] = self.desc.attr(item)
+
             if self._has_kernel(type):
                 self.desc.infer_var_type(self.block.desc)
                 self.desc.infer_shape(self.block.desc)
 
     def _has_kernel(self, op_type):
         return op_type not in self.OP_WITHOUT_KERNEL_SET
 
+    def _get_runtime_attrs(self):
```

Comment:
Add comments to avoid other one using this.

Reply:
done